src/coreclr/gc/gcinterface.h

```diff
@@ -932,6 +932,9 @@ class IGCHeap {
     // Walk the heap object by object.
     virtual void DiagWalkHeap(walk_fn fn, void* context, int gen_number, bool walk_large_object_heap_p) PURE_VIRTUAL
 
+    // Walk the heap object by object outside of a GC.
+    virtual void DiagWalkHeapStandalone(walk_fn fn, void* context, int gen_number, bool walk_large_object_heap_p) PURE_VIRTUAL
```

Comment:
- Bump the GC interface version https://github.com/dotnet/runtime/blob/c40ecbc6d2fd94a0e9eec5b0154899aa3458dce9/src/coreclr/gc/gcinterface.h#L7-L14

- I think you want to add this method to the very end to make the change non-breaking (minor version bump only)

Reply:
Thank you for the catch!